client/app/email-domain/general-informations/email-domain-general-informations.html

```diff
@@ -1,125 +1,105 @@
+                            <li data-ng-repeat="mxRecord in ctrlTabGeneralInformations.mxRecords track by $index"
+                                data-ng-bind="mxRecord">
+                            </li>
+                        </ul>
+                        <span data-ng-if="!ctrlTabGeneralInformations.mxRecords.length">-</span>
+                    </oui-tile-description>
+                </oui-tile-definition>
+            </oui-tile>
+        </div>
+        <div class="col-sm-4 mb-5">
+            <oui-tile data-heading="{{ ::'email_dashboard_quotas' | translate }}"
+                      data-loading="ctrlTabGeneralInformations.loading.quotas"
+                      class="h-100">
+                    <oui-tile-definition data-term="{{ ::'email_tab_accounts_quota' | translate }}"
+                                         data-term-popover="{{ ctrlTabGeneralInformations.domain.offer !== 'redirect'? ('email_tab_accounts_quota_info' | translate): '' }}"
+                                         data-description="{{ (ctrlTabGeneralInformations.summary.account || '0') + ' / ' + (ctrlTabGeneralInformations.quotas.account || '0') }}"></oui-tile-definition>
+                    <oui-tile-definition data-term="{{ ::'email_tab_responders_quota' | translate }}"
+                                         data-description="{{ (ctrlTabGeneralInformations.summary.responder || '0') + ' / ' + (ctrlTabGeneralInformations.quotas.responder || '0') }}"></oui-tile-definition>
+                    <oui-tile-definition data-term="{{ ::'email_tab_redirections_quota' | translate }}"
+                                         data-description="{{ (ctrlTabGeneralInformations.summary.redirection || '0') + ' / ' + ((ctrlTabGeneralInformations.quotas.redirection + ctrlTabGeneralInformations.quotas.alias) || '0') }}"></oui-tile-definition>
+            </oui-tile>
+        </div>
+        <div class="col-sm-4 mb-5">
+            <oui-tile data-heading="{{ ::'common_plan' | translate }}"
+                      data-loading="ctrlTabGeneralInformations.loading.domain || ctrlTabGeneralInformations.loading.serviceInfos"
+                      class="h-100">
+                <oui-tile-definition data-term="{{ ::'email_tab_domain_offer' | translate }}"
+                                     data-description="{{ ctrlTabGeneralInformations.domain.offer || '-' }}">
+                    <oui-action-menu data-compact align="end">
+                        <oui-action-menu-item data-on-click="ctrlTabGeneralInformations.gotoMxPlans()">
+                            <span data-translate="common_upgrade_account"></span>
+                        </oui-action-menu-item>
+                    </oui-action-menu>
+                </oui-tile-definition>
+                <oui-tile-definition data-term="{{ ::'contacts_management'|translate }}">
+                    <oui-tile-description>
+                        <div>
```

Comment:
Looks more like a `p`, maybe you can change `global_admin ` to have everything in a `p`